microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -35,25 +67,37 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
 <!-- ====================================================================== -->
 ## Before you begin
 
-To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials. Use the following steps to create API credentials.
+To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials.  Use the following steps to create API credentials.
 
 1. Visit Microsoft Partner Center and sign in to the account that you used to publish an add-on.
 
 1. Under the **Microsoft Edge** program, select **Publish API**.
```

Comment:
is there a UI control in Partner Center page to select v1 vs v2 API?  
@bhuvanapriyap @yvsubhash